mapslice_test.go

```diff
@@ -50,3 +50,33 @@ func TestUnmarshal(t *testing.T) {
 		t.Errorf("expected: %s\ngot: %s", e, r)
 	}
 }
+
+func ExampleMapSlice_MarshalJSON() {
+	ms := MapSlice{
+		MapItem{"abc", 123, 0},
+		MapItem{"def", 456, 0},
+		MapItem{"ghi", 789, 0},
+	}
+
+	b, err := json.Marshal(ms)
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	fmt.Printf("%s", b)
+
+	// Output:
+	// {"abc":123,"def":456,"ghi":789}
+}
+
+func ExampleMapSlice_UnmarshalJSON() {
+	var ms = MapSlice{}
+
+	if err := json.Unmarshal([]byte(`{"abc":123,"def":456,"ghi":789}`), &ms); err != nil {
+		fmt.Println(err)
```

Comment:
Curious why you choose to use `fmt.Println()` here rather than `Printf` as above. The above would require a `string(b)` but consistency is preferable.